runtime/caml/stack.h

```diff
@@ -100,11 +100,8 @@ typedef struct {
     num_debug is num_alloc if frame_size & 2, otherwise 1. */
 } frame_descr;
 
-static inline uintnat Wosize_encoded_alloc_len(unsigned char n)
-{
-  /* Allocation lengths are encoded as 0-255, giving sizes 1-256 */
-  return n + 1;
-}
+/* Allocation lengths are encoded as 0-255, giving sizes 1-256 */
+#define Wosize_encoded_alloc_len(n) ((n) + 1)
```

Comment:
You need to cast `n` to `uintnat` to avoid overflows. `n` is typically an `unsigned char`. (Yes, I know that according to the C standard, these computations are done in `int`. But do you really want to rely on this?)

Reply:
Changed for readability. (I don't know of compilers that get this wrong, though, if that's what you're worried about)